tests/app/clients/test_aws_pinpoint.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from app import aws_pinpoint_client
+from tests.conftest import set_config_values
+
+
+@pytest.mark.serial
```

Comment:
There's some sort of race condition in our tests that I couldn't figure out :/ If these two tests are not run serially then something resets the config variables that they change.

Reply:
It would be a great topic for a code review session. It seems we have a misunderstanding on our some of our test concurrency is setup and fails when they should be successful. We can put our mind to it and try to brainstorm on this.

Reply:
I gave some more thoughts to this and one thing I wish we could do better  is OOP in our code base. We barely have much and this could be our focus on code reviews session. We rely a lot on environment variables which is not a good practice in general, as the code relies on environment rather than configuration of the logic to run. Encapsulating the pool config in an object would make this easier probably to run instead of relying on environments. Do you know what I mean here?

Reply:
We do pull the env vars out of the env and put into the `Config` object, but we should make sure that we always read that object and never the env vars directly (which I think we're pretty good at?). I think that there's magic going on behind the scenes though (with flask, for example), where it takes vars directly from the env :/

In this case, though, in the test I made serial the app wasn't using the values of `Config` that I added in the test setup 😞 Which - how can that happen?